### ISSUE TITLE
Error: Invalid IPC sender

### ISSUE THREAD

**Issue by EnviralDesign** [whitphx/stlite#969]
Greetings, I am encountering this error:
![image](https://github.com/whitphx/stlite/assets/10091486/04fef12f-a191-47e8-80b9-745a2a8178cc)

Steps to reproduce:
1) Launch main app page, which is just a landing page with side bar navigation open.
2) I click on any initial page, doesn't seem to matter which - **this loads fine**.
3) I click on any subsequent page, doesn't seem to matter which - **this creates the above error**.

Additional Notes:

This does not happen when running the app directly with `streamlit run app.py`

This happens specifically when testing the build using `npm run serve` after running `npm run dump`.

Here's my package.json:
```json

{
  "name": "efx_dashboard",
  "version": "0.1.0",
  "main": "./build/electron/main.js",
  "scripts": {
    "dump": "dump-stlite-desktop-artifacts",
    "serve": "cross-env NODE_ENV=production electron .",
    "pack": "electron-builder --dir",
    "dist": "electron-builder",
    "postinstall": "electron-builder install-app-deps"
  },
  "build": {
    "files": [
      "build/**/*",
      "data/**/*",
      ".streamlit/**/*"
    ],
    "directories": {
      "buildResources": "assets"
    },
    "win": {
      "target": "portable"
    }
  },
  "devDependencies": {
    "@stlite/desktop": "^0.58.3",
    "cross-env": "^7.0.3",
    "electron": "30.0.8",
    "electron-builder": "^24.13.3"
  },
  "stlite": {
    "desktop": {
      "files": [
        "app.py",
        "menu.py",
        "utils.py",
        "pages/*.py",
        "data/*",
        ".streamlit/*"
      ],
      "requirementsTxtFiles": ["requirements.txt"],
      "embed": true,
      "nodeJsWorker": true,
      "nodefsMountpoints": {
        "/mnt": "."
      },
      "entrypoint": "app.py"
    }
  }
}

```

**Comment by whitphx** [comment]
Thank you for this report! This should be fixed in the next release, 0.58.4.

**Comment by EnviralDesign** [comment]
Thank you!!
